README.rst

```diff
@@ -803,6 +811,67 @@ Mobile Access Technology in use
 .. figure:: https://github.com/openwisp/openwisp-monitoring/raw/docs/docs/access-technology.png
   :align: center
 
+Iperf
+~~~~~
+
++--------------------+---------------------------------------------------------------------------------------------------------------------------+
+| **measurement**:   | ``iperf``                                                                                                                 |
++--------------------+---------------------------------------------------------------------------------------------------------------------------+
+| **types**:         | | ``int`` (iperf_result, sent_bytes_tcp, received_bytes_tcp, retransmits, sent_bytes_udp, total_packets, lost_packets),   |
+|                    | | ``float`` (sent_bps_tcp, received_bps_tcp, sent_bps_udp, jitter, lost_percent)                                          |
++--------------------+---------------------------------------------------------------------------------------------------------------------------+
+| **fields**:        | | ``iperf_result``, ``sent_bps_tcp``, ``received_bps_tcp``, ``sent_bytes_tcp``, ``received_bytes_tcp``, ``retransmits``,  |
+|                    | | ``sent_bps_udp``, ``sent_bytes_udp``, ``jitter``, ``total_packets``, ``lost_packets``, ``lost_percent``                 |
++--------------------+---------------------------------------------------------------------------------------------------------------------------+
+| **configuration**: | ``iperf``                                                                                                                 |
++--------------------+---------------------------------------------------------------------------------------------------------------------------+
+| **charts**:        | | ``bandwidth_tcp``, ``transfer_tcp``, ``retransmits``, ``bandwidth_udp``                                                 |
+|                    | | ``transfer_udp``, ``jitter``, ``datagram``, ``datagram_loss``                                                           |
++--------------------+---------------------------------------------------------------------------------------------------------------------------+
+
+**Bandwidth (TCP)**:
+
+.. figure:: https://github.com/openwisp/openwisp-monitoring/raw/docs/docs/1.1/bandwidth-tcp.png
+  :align: center
+
+**Transfer (TCP)**:
+
+.. figure:: https://github.com/openwisp/openwisp-monitoring/raw/docs/docs/1.1/transfer-tcp.png
```

Comment:
let's do those changes to the charts that we discussed (have bandwidth and transfer as simple lines with no fill and have 2 lines in each chart, one for TCP and one for UDP, then update the README here and screenshots.